dapps/profile.vue/src/components/verifications/overview/overview.ts

```diff
@@ -63,7 +63,7 @@ export default class VerificationsOverviewComponent extends mixins(EvanComponent
     this.address = this.$route.params.address || runtime.activeIdentity;
 
     // switch issue account
```

Comment:
Is this comment outdated? What does the hardcoded string refer to?

Reply:
Good question. I think we can remove it.